dcrlibwallet.go

```diff
@@ -96,12 +96,33 @@ func newLibWallet(walletDataDir, walletDbDriver string, activeNet *netparams.Par
 		go shutdownListener()
 	}
 
+	// Init Sync Data
```

Comment:
Probably best to do this when sync starts instead of here - `spvSync` and `rpcSync` functions.